packages/SqueakMaps-TiledMaps.package/SMATiledMapMorph.class/instance/locate.and..st

```diff
@@ -0,0 +1,25 @@
+locating
+locate: aQueryString and: anotherQueryString
```

Comment:
Umbenennen zu `locateLocations: aQueryString and: anotherQueryString` damit klarer wird, dass es mehrere Locations sind im Gegensatz zu nur `locate`?

Reply:
Dann sind die Signaturen verschieden, obwohl sie das gleiche machen. Einmal einen Ort locaten und einmal zwei Orte. Das zwei Parameter akzeptiert werden, war in unseren Augen genug, um den Unterschied klar zu machen, bei gleichzeitigem Erhalt der Gemeinsamkeit. Aber wir könnten darüber reden, ob wir die Methode komplett anders nennen, wenn wir die Routing Funktionalität eingebaut haben. Weil sie located ja nicht mehr nur 2 Punkte wie ursprünglich geplant, als die Methode entstand, sondern sucht auch die Route raus und zeigt sie an zwischen den beiden Orten. Ich würde in diesem merge den Namen aber nochmal lassen wie er grade ist.

Reply:
Ja stimm ich dir zu. In Zukunft sollte `locate` wahrscheinlich einfach nur eine Stringanfrage in Geokoordinaten umwandeln, damit die dann in die RoutingAPI geparst werden können. Dann könnte man auch das setzen der Pins unabhängig von `locate` machen, sodass nicht immer durch `locate` auch Pins gesetzt werden.